app/Console/Commands/Randomize.php

```diff
@@ -124,6 +125,10 @@ public function handle()
                 return 3;
             }
 
+            if (is_string($this->option('heartcolor'))) {
+                $rom->setHeartColors($this->option('heartcolor'));
```

Comment:
To my other comment this is where I would expect a random pick for heart color (and probably use `get_random_int` function as it will use csprng)